frontend/assets/javascripts/src/modules/videoOverlay.js

```diff
@@ -27,36 +27,66 @@ define(['src/modules/raven'],function(raven) {
             var playerApi;
 
             if(playerIframe && playerOverlay) {
-                playerApi = new YT.Player(playerIframe, {
-                    events: {
-                        'onReady': function() {
-                            playerReady(player, playerApi, playerOverlay);
+
+                var autoplay = playerIframe.classList.contains('autoplay');
+                var loop = playerIframe.classList.contains('autoplay');
```

Comment:
Yup, it definitely should 😮.

In fact the use of `classList` will break in older versions of IE. I know that's a small chunk of the audience, but I've added a workaround just to be safe.

42f3de0